.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,25 @@
+name: FrontEnd CICD
+
+on: [pull_request]
```

Comment:
How about branches, is it necessary ? (source: **https://github.com/readme/guides/sothebys-github-actions**) 
on:
  pull_request:
    branches:
      - main  # Adjust this to your default branch

Reply:
Now, I focus to clean and test in all branches. Later i will do for specific branch, like building and deploying for main branch.